Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/signature/core/DscRepository.kt

```diff
@@ -58,6 +58,9 @@ class DscRepository @Inject constructor(
     suspend fun clear() {
         Timber.d("clear()")
         localStorage.clear()
+        internalData.updateBlocking {
+            defaultDscData.getDscData()
+        }
```

Comment:
why are you adding the default value again ?

Reply:
Because even if we clear the cache, `internalData` flow is not updated immediately by default